src/modules/StorageProposal.rb

```diff
@@ -6300,8 +6280,9 @@ def CommonWidgetsHelp()
       # TRANSLATORS: help text
       help_text =
         _(
-          "<p>To create an LVM-based proposal, choose the corresponding button. The\n" +
-          "LVM-based proposal can be encrypted.</p>\n"
+          "<p>Choose Partition-based Proposal if you don't want to use LVM.\n" +
```

Comment:
NP: IIRC usually the  button labels are emphasized in the help text so it's obvious it's reference to the UI, e.g. `Choose <b>Partition-based Proposal</b> if ...`